client/components/auth-form.jsx

```diff
@@ -36,11 +32,7 @@ export default class AuthForm extends React.Component {
       .then(res => res.json())
       .then(result => {
       });
-    this.setState({ username: '', password: '', region: '', timeAvailable: '' });
-    this.userRef.current.value = '';
-    this.passRef.current.value = '';
-    this.timeRef.current.value = 'placeholder';
-    this.regRef.current.value = 'placeholder';
+    event.target.reset();
```

Comment:
The best way to reset a form is to set the state properties to what you want and let the component re-render with the empty values. Your line 39 was right.